.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "mshadow"]
 	path = mshadow
-	url = https://github.com/dmlc/mshadow.git
+	url = https://github.com/larroy/mshadow.git
 [submodule "dmlc-core"]
 	path = dmlc-core
-	url = https://github.com/dmlc/dmlc-core.git
+	url = https://github.com/larroy/dmlc-core.git
```

Comment:
This is probably for testing. Leaving a note here just to make sure this change doesn't get accidentally merged.

Reply:
Please ping me when the submodules are merged accordingly so we can rerun the build with the right submodule

Reply:
🤦‍♂️